pkg/camo/proxy.go

```diff
@@ -246,7 +248,11 @@ func (p *Proxy) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 		if mlog.HasDebug() {
 			mlog.Debugx("content length exceeded", mlog.A("url", sURL))
 		}
-		http.Error(w, "Content length exceeded", http.StatusNotFound)
+		if p.config.MaxSizeRedirect != "" {
+			http.Redirect(w, req, p.config.MaxSizeRedirect, http.StatusFound)
```

Comment:
I would expect to see a 302 or 303 redirect status not a 404. Does using a 404 here work well for you in practice?

Reply:
ah. nevermind. I just read the diff wrong the second time.